demos/desktop-chat-app/frontend/src/components/CurrentUserAvatar/index.jsx

```diff
@@ -20,7 +20,7 @@ function CurrentUserAvatar({ className }) {
     if (registry && registry.users && registry.users[nodeAddress]) {
         username = registry.users[nodeAddress].username
         if (registry.users[nodeAddress].photo) {
-            userPhotoURL = `http://localhost:8080/users/${nodeAddress}/photo?state_uri=${selectedServer}/registry`
+            userPhotoURL = `http://localhost:8080/users/${nodeAddress}/photo?state_uri=${selectedServer}/registry&${Date.now()}`
```

Comment:
```suggestion
            userPhotoURL = `http://localhost:8080/users/${nodeAddress}/photo?state_uri=${encodeURIComponent(selectedServer)}/registry&${Date.now()}`
```
This bit me the other day when I tried to create a room called `chat.redwood.dev/#general`